ini.go

```diff
@@ -134,6 +136,12 @@ func LoadSources(opts LoadOptions, source interface{}, others ...interface{}) (_
 	return f, nil
 }
 
+func LoadCustomEmpty(opts LoadOptions) *File {
```

Comment:
I think it's cleaner to just change the following function to accept an optional `LoadOptions` via variable-length arguments, i.e. 

```go
func Empty(opts ... LoadOptions) *File {
	var opt LoadOptions
	if len(opts) > 0 {
		opt = opts[0]
	}

	...
}
```

https://github.com/go-ini/ini/blob/dd614b817be88f32a238e94e84f11da890471a7c/file.go#L61

Reply:
Yes, good idea